CONTRIBUTING.md

```diff
@@ -21,35 +21,39 @@ pull requests.
   client for the language server
 * `test-workspace` directory for manually testing the plugin locally
 
-## Running locally
-
-First you need to have the following applications installed
+## Running a local version of the server
+You will need the following applications installed:
 
 * `git`
 * `sbt`
 * Visual Studio Code (`code` from the console)
-* `npm`
-
-To try out metals locally, it's best to keep two open terminal windows.
 
 ```
-git clone https://github.com/scalameta/metals.git
-cd metals
-
-########
-# Step 1
-########
 sbt
-> metals/publishLocal # publish your latest changes locally
+> metals/publishLocal     # publish your latest changes locally
                           # keep this sbt shell session open, and
                           # re-run publishLocal every time you
                           # edit *.scala sources.
 > ~testWorkspace/test:compile # compile the sources in test-workspace
+```
+
+You can then use the Metals VSCode extension published on the Marketplace, and point it to the local
+snapshot version of the server you've just published by changing the settings:
 
-########
-# Step 2
-########
-# Inside a new terminal window
+```json
+"metals.serverVersion": "0.2.0-SNAPSHOT"
+```
+
+Then open the metals root with VSCode (`code test-workspace` from the console) and try your changes.
```

Comment:
"open test workspace"? it's not metals root, right?

Reply:
right, I've changed this mid-sentence